clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -762,6 +769,13 @@ def _strip_storage_protocol(storage_type: StorageType, path: str):
         if storage_type == StorageType.LOCAL:
             return path[len(f"{StorageType.LOCAL.value}:") :]
 
+        # OSS has different behavior than S3 and GCS, if we do not remove the
+        # protocol, it will always return an empty array.
+        if storage_type == StorageType.OSS:
```

Comment:
It's better add the `uri` example in the method annotation.

Reply:
Yeah, please explain more.

Reply:
Added.